app/admin/user.rb

```diff
@@ -99,6 +99,9 @@
           end
           row :skills
           row :mentor
+          row :mentees do |user|
+            User.mentees_for(user)
```

Comment:
why not a relation and call: `user.mentees` ?

Reply:
My bad, I assumed we didn't wanted to interfere with any relations. Just updated the PR to use a `has_many` relationship instead.